webgpu.h

```diff
@@ -1155,44 +1155,149 @@ typedef enum WGPUWaitStatus {
  *
  * @{
  */
+/**
+ * For reserved non-standard bitflag values, see @ref BitflagRegistry.
+ */
 typedef WGPUFlags WGPUBufferUsage;
+/**
```

Comment:
nit. These all seem really redundant, given that its documented in the code right below. Is it for doc generation that this is necessary?

Reply:
You're right it's pretty redundant. Turns out the docs do show the definition:
![image](https://github.com/user-attachments/assets/76bb7a70-7550-4873-ae41-051a7c54c6b5)
unlike for enums:
![image](https://github.com/user-attachments/assets/7ffa623a-d8df-4f05-88ac-d953db83caa7)

Of course this html view is not the only possible way to look at things. But maybe the only useful docs are special cases like 0 and Red|Green|Blue|Alpha. Will follow up.